nodestream_akamai/gtm/gtm.py

```diff
@@ -2,28 +2,44 @@
 
 from nodestream.pipeline.extractors import Extractor
 
-from ..akamai_utils.gtm_client import AkamaiGTMClient
+from ..akamai_utils.gtm_client import AkamaiGtmClient
 
 
-class AkamaiGTMExtractor(Extractor):
+class AkamaiGtmExtractor(Extractor):
     def __init__(self, **akamai_client_kwargs) -> None:
-        self.client = AkamaiGTMClient(**akamai_client_kwargs)
+        self.client = AkamaiGtmClient(**akamai_client_kwargs)
         self.logger = logging.getLogger(self.__class__.__name__)
 
     async def extract_records(self):
         for domain in self.client.list_gtm_domains():
             try:
                 raw_domain = self.client.get_gtm_domain(domain["name"])
-                summary = []
+                deeplink = [
+                    link for link in raw_domain["links"] if link["rel"] == "self"
+                ][0]["href"]
+                properties = []
                 for property in raw_domain["properties"]:
                     fqdn = property["name"] + "." + raw_domain["name"]
-                    summary.append(
-                        {"fqdn": fqdn, "trafficTargets": property["trafficTargets"]}
+                    servers = []
+                    for traffic_target in property["trafficTargets"]:
+                        servers.extend(traffic_target["servers"])
+                    properties.append(
+                        {
+                            "fqdn": fqdn,
+                            "type": property["type"],
+                            "name": property["name"],
+                            "trafficTargets": servers,
+                        }
                     )
                 parsed_domain = {
                     "name": raw_domain["name"],
-                    "summary": summary,
-                    "raw": raw_domain,
+                    "properties": properties,
+                    "type": raw_domain["type"],
+                    "loadImbalancePercentage": raw_domain["loadImbalancePercentage"],
+                    "loadFeedback": raw_domain["loadFeedback"],
+                    "cnameCoalescingEnabled": raw_domain["cnameCoalescingEnabled"],
+                    "deeplink": deeplink
+                    # "raw": raw_domain,
```

Comment:
Can this be removed?

Reply:
Why? These were just additional properties I felt might be useful in the GTM_DOMAIN object

Reply:
I just meant the comment. The properties are all good 👍

Reply:
Ah I see. Yes, lemme pull that